simulation/setup_swarm.cpp

```diff
@@ -395,13 +395,14 @@ void setup_swarm(int num_nodes
 
 		bool shut_down = terminate(tick, *nodes[0]);
 
-		if (type == swarm_test::upload)
+		if (type == swarm_test::upload || type == swarm_test::upload_slow)
```

Comment:
as my comments may have suggested. It's not obvious to me exactly what this test mode does or why it's necessary. I get the impression there may be a better name for it. Is the main difference that this doesn't automatically end the test when the other peers are seeds?